mesheryctl/internal/cli/root/mesh/remove.go

```diff
@@ -70,4 +81,52 @@ func init() {
 	removeCmd.Flags().StringVarP(&namespace, "namespace", "n", "default", "Kubernetes namespace to be used for deploying the validation tests and sample workload")
 	removeCmd.Flags().StringVarP(&tokenPath, "tokenPath", "t", "", "Path to token for authenticating to Meshery API")
 	_ = removeCmd.MarkFlagRequired("tokenPath")
+	removeCmd.Flags().BoolVarP(&watch, "watch", "w", false, "Watch for events and verify operation (in beta testing)")
+}
+
+func waitForRemoveResponse(mctlCfg *config.MesheryCtlConfig, query string) (string, error) {
+	path := mctlCfg.GetBaseMesheryURL() + "/api/events?client=cli_remove"
```

Comment:
@leecalcote @DelusionalOptimist I checked with watch flag to check if this function works... But it threw error (of course mesh is removed but GET request threw error while testing). Is the API call URL correct?

Reply:
@mgfeller with respect to the upcoming focus on Messaging Framework, it'd be good to familiarize with the /api/events. Gosh, it's been so long that I'm not sure what the expected behavior here is.

Reply:
I referred the REST API docs too to get clarified https://docs.meshery.io/reference/rest-apis ... But no luck, maybe I used a wrong URL by mistake 😬

Reply:
@piyushsingariya might assist here.

Reply:
This URL isn't covered in the docs, will check if the URL is correct and get back here soon.

Reply:
@piyushsingariya is the URL thing solved? // Just pinging for update

Reply:
URL is correct. You must be getting any errors if it's not 200. Could you paste that here?

Reply:
I'll check again and get back to you...

Reply:
Not error codes... but GET request error...
![image](https://user-images.githubusercontent.com/59508546/135967187-c7644ca1-03f1-4fe8-aec2-003091d1beb3.png)

Reply:
And it pauses after that, ending in infinite loop....

Reply:
@utkarsh-pro Help!!